openedx/core/djangoapps/content/course_overviews/serializers.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+CourseOverview serializers
+"""
+from rest_framework import serializers
+
+from lms.djangoapps.certificates.api import get_certificate_for_user
+from lms.djangoapps.program_enrollments.api.api import (
+    get_due_dates,
+    get_course_run_url,
+    get_emails_enabled,
+    get_course_run_status
+)
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.helpers import get_resume_urls_for_enrollments
+from student.models import CourseEnrollment
+
+
+class CourseOverviewBaseSerializer(serializers.ModelSerializer):
+    """
+    Serializer for a course run overview.
+    """
+
+    class Meta(object):
+        model = CourseOverview
+        fields = (
+            'id',
+            'start',
+            'end',
+        )
+
+
+class CourseOverviewRequestUserSerializer(CourseOverviewBaseSerializer):
```

Comment:
I would move this out to the djangoapp that lives in edx-enterprise. The `get_course_overviews` function should just return the serialized vanilla CourseOverview models. Let the learner portal specific code add in the additional information it needs.